backend/src/main/java/com/example/skicurort/curort/CurortControler.java

```diff
@@ -0,0 +1,57 @@
+package com.example.skicurort.curort;
+
+import static com.example.skicurort.curort.CurortMapper.*;
+
+import com.example.skicurort.exception.NoIdException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
zamiast importować wszystkie klasy najlepiej zaimportować te, które są używane. Organize imoprt w inteliJ powinien to załatwić.

Reply:
Czy chodzi tu o ...bind.annotation.* bo to wchodzi mi z automatu jak z jednego importu wiecej niż 4 klasy zaciagam to zmienia na ... .*. Generalnie tak mi intelij to ogarnia po Optimize imports

Reply:
https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports link jak wyłączyć wildcard w inteliJ

Reply:
ok zmieniłem